RogueEssence/Menu/Items/ItemMenu.cs

```diff
@@ -130,6 +130,22 @@ private InvItem getChosenItemID(int menuIndex)
 
         protected override void UpdateKeys(InputManager input)
         {
+            if (GameManager.Instance.CurrentScene == DungeonScene.Instance)
```

Comment:
Re-order the checks such that the input check goes first.  That way, you make DungeonScene checks and computations less often.